udata_front/theme/gouvfr/templates/svg/chevron.svg

```diff
@@ -1,3 +1,3 @@
 <svg aria-hidden="true" width="6" height="9" viewBox="0 0 6 9" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M5.25 4.89844C5.46094 4.6875 5.46094 4.33594 5.25 4.10156L2.0625 0.914062C1.82812 0.703125 1.47656 0.703125 1.26562 0.914062L0.726562 1.45312C0.515625 1.6875 0.515625 2.03906 0.726562 2.25L3 4.52344L0.726562 6.77344C0.515625 6.98438 0.515625 7.33594 0.726562 7.57031L1.26562 8.08594C1.47656 8.32031 1.82812 8.32031 2.0625 8.08594L5.25 4.89844Z" fill="#198CF2"/>
```

Comment:
I guess we use chevron only in some places left, we'll be able to remove it later :)

Reply:
I added the DSFR icon in post/display, it was the last one to use this svg.